src/visualization.jl

```diff
@@ -35,22 +38,22 @@ end
 
 # Examples
 ```julia
-scene = Scene(500)
-get!(scene,traj_interaction,1);
-scene_list = get_hallucination_scenes(scene,models=models);
-scenelist2video(scene_list,filename="media/scenelist_to_video.mp4")
+# See run_vehicles function in driving_simulation.jl
 ```
 """
 function scenelist2video(scene_list;id_list=[],roadway,filename)
+    AutomotiveVisualization.set_render_mode(:fancy)
     frames = Frames(MIME("image/png"),fps = 10)
-    
+    mr = MergingRoadway(roadway) # Wrapper for specialized render for merging lanes
+    mp = VecE2(1064.5227, 959.1559)
```

Comment:
maybe store this in a constant in the main scope of the package instead of hardcoding it in the function.

Reply:
especially since you are reusing it at several places

Reply:
Removed the set_render_mode("fancy) and instead used `FancyCar` within the creation of `renderables`.